### PR TITLE
adding SugoiAxiLitePixelMatrixConfig FW/SW

### PR DESCRIPTION
**Pull request by ruck314**
### Description
- https://confluence.slac.stanford.edu/display/ppareg/Pixel+matrix%3A+in-pixel+configuration
- Used to configure pixel array via AXI-Lite instead the ASIC

### CODE REVIEW

**Comment by ruck314** [comment]
@king-pietro What's the status update of this effort? Do you think it is ready for merging into pre-release?

**Comment by king-pietro** [comment]
> @king-pietro What's the status update of this effort? Do you think it is ready for merging into pre-release?

I tested it and it works fine. I wanted to wait until the tapeout of the ASIC for the merge in case of last minute changes, but I guess we could merge.

**Comment by ruck314** [comment]
@king-pietro I move from draft to review ready.  Once you are ready, go ahead and approve then merge the PR

**Review by king-pietro — APPROVED** [review]

**Pull request merged**
